client.go

```diff
@@ -83,6 +87,21 @@ func CreateUnified() (Interface, error) {
 	return client, nil
 }
 
+// CreateUnified returns the unified client
+func CreateUnifiedWithOptions(opts *Options) (Interface, error) {
```

Comment:
Is it possible to create client via CreateUnified() and then modify its rest.Config?

Reply:
Alternatively we can also consider using functional options, see https://golang.cafe/blog/golang-functional-options-pattern.html

Reply:
no, it returns the clientset at this level we only can call resources.